MekHQ/src/mekhq/campaign/ai/utility/HasMemory.java

```diff
@@ -0,0 +1,7 @@
+package mekhq.campaign.ai.utility;
```

Comment:
Don't forget the legal notice on your new classes. You're missing them in a couple of places. I know this is still WIP, so you might not have got around to them yet, but figured it was worth mentioning just in case.